roles/core/dns_server/tasks/master.yml

```diff
@@ -0,0 +1,64 @@
+---
+- name: Template >> Forward zone file on Master DNS
+  template:
+    src: forward.j2
+    dest: /var/named/forward
+    owner: root
+    group: root
+    mode: 0644
+  notify: Restart dns services
+  register: zoneconf
+  when: "dns_master in ansible_all_ipv4_addresses"
```

Comment:
The file `roles/core/dns_server/tasks/master.yml` is executed only if "dns_master in ansible_all_ipv4_addresses" in  `roles/core/dns_server/tasks/main.yml`.
Isn't this condition redundant ?

Reply:
Yes. I'll double check.

Reply:
Good catch. At first these tasks were in the main.yml, then I moved them to the master.yml but forgot to remove the when clause. I'll update the code.